tests/test_worker.py

```diff
@@ -196,7 +196,7 @@ def _mock_pods_stream_that_returns_running_pod(*args, **kwargs):
             command="python -m prefect.engine",
             env={
                 **get_current_settings().to_environment_variables(exclude_unset=True),
-                "PREFECT__FLOW_RUN_ID": flow_run.id.hex,
+                "PREFECT__FLOW_RUN_ID": str(flow_run.id),
```

Comment:
These test changes account for a change in the format of the `PREFECT__FLOW_RUN_ID` in `prefect`.